forge-gui/res/cardsfolder/u/undercover_butler.txt

```diff
@@ -0,0 +1,8 @@
+Name:Undercover Butler
+ManaCost: 2 UB
```

Comment:
There's an extra space after the : here. Most likely not functionally relevant, just a matter of style (would be good to remove for consistency)

Reply:
Oh, that was an editing lapse of mine. I'll be sure to double check going forward and to fix this one on the next pull request.